src/util/ui_message.cpp

```diff
@@ -57,7 +61,19 @@ ui_message_handlert::ui_message_handlert(
     cmdline.isset("xml-ui")?uit::XML_UI:
     cmdline.isset("json-ui")?uit::JSON_UI:
     uit::PLAIN,
-    program)
+    program,
+#ifndef _WIN32
+    cmdline.isset("timestamp")
+      ? cmdline.get_value("timestamp") == "monotonic"
+          ? timestampert::clockt::MONOTONIC
+          : cmdline.get_value("timestamp") == "wall"
+              ? timestampert::clockt::WALL_CLOCK
+              : timestampert::clockt::NONE
+      : timestampert::clockt::NONE
+#else
+      timestampert::clockt::NONE
+#endif
```

Comment:
I don't think you need the `#ifndef _WIN32` here: for `_WIN32`, your code simply ignores the `clock_type` argument, and hence you're completely safe to us the full ternary branch above in either case!?